### ISSUE TITLE
BBC One South West icon missing

### ISSUE THREAD

**Issue by CaptainTK** [vonH/plugin.video.iplayerwww#142]
@primaeval, could you do me a favour, please? I am struggling to create a new icon for BBC One South West. I do have the Gill Sans font, but it seems to be slightly different from yours. When I create the image with your code in ImageMagick, I get text, which is smaller. If I tweak the pointsize to get the right size, I get text, which is too wide. This will look strange if you scroll through the channels and I would like to avoid recreating all icons for this.

Long story short: Could you please generate the icon for BBC One South West?

**Comment by primaeval** [comment]
Here you go. Here is the font and script too. I've made another one somewhere that uses the right colours. I'll see if I can find that later.
![bbc_one_south_west](https://cloud.githubusercontent.com/assets/11716348/23449396/9c523c56-fe56-11e6-89f3-5d3e0a49c0cc.png)

[GIL_____.zip](https://github.com/vonH/plugin.video.iplayerwww/files/809908/GIL_____.zip)

[tv_icons.zip](https://github.com/vonH/plugin.video.iplayerwww/files/809912/tv_icons.zip)

**Comment by CaptainTK** [comment]
Splendid, thank you very much @primaeval!
